No colord (needs GUdev) on OpenBSD (XXX add HAVE_COLORD to meson_options.txt)

Index: src/meson.build
--- src/meson.build.orig
+++ src/meson.build
@@ -22,7 +22,6 @@ mutter_pkg_deps = [
 ]
 
 mutter_pkg_private_deps = [
-  colord_dep,
   lcms2_dep,
   gmodule_no_export_dep,
   gnome_settings_daemon_dep,
@@ -188,15 +187,6 @@ mutter_sources = [
   'backends/meta-backend-private.h',
   'backends/meta-barrier.c',
   'backends/meta-barrier-private.h',
-  'backends/meta-color-device.c',
-  'backends/meta-color-device.h',
-  'backends/meta-color-manager.c',
-  'backends/meta-color-manager.h',
-  'backends/meta-color-manager-private.h',
-  'backends/meta-color-profile.c',
-  'backends/meta-color-profile.h',
-  'backends/meta-color-store.c',
-  'backends/meta-color-store.h',
   'backends/meta-crtc-mode.c',
   'backends/meta-crtc-mode.h',
   'backends/meta-crtc.c',
@@ -392,8 +382,6 @@ if have_x11
     'backends/x11/meta-barrier-x11.h',
     'backends/x11/meta-clutter-backend-x11.c',
     'backends/x11/meta-clutter-backend-x11.h',
-    'backends/x11/meta-color-manager-x11.c',
-    'backends/x11/meta-color-manager-x11.h',
     'backends/x11/meta-crtc-xrandr.c',
     'backends/x11/meta-crtc-xrandr.h',
     'backends/x11/meta-cursor-renderer-x11.c',
